config/rbac/auth_proxy_service.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   ports:
   - name: https
-    port: 9289
-    targetPort: 9289
+    port: 8443
+    targetPort: https
```

Comment:
shouldn't this one be `9289` instead of `https`.  `https` is defined with port 8443. 
I am unable to get any metrics if its `https` port, where as `9289` i am able to do curl and get all the metrics. i am assuming this because of `--upstream`   in manager-auth-proxy-patch.yaml